aroma/plotting.py

```
@@ -23,15 +23,23 @@ def classification_plot(in_file, out_dir):
     out_dir : str
         Output directory.
     """
+    try:
+        package = 'seaborn'
+        __import__(package)
+    except ImportError:
+        LGR.warning('Seaborn is needed for plotting, isntalling')
```

```suggestion
        LGR.warning('Seaborn is needed for plotting, installing')
```

---

This is a very cool trick, but I don't think it's a good idea to try to install something automatically. I would just raise an Exception and point to the plotting-specific installation instructions.

---

That's what I was thinking. Thanks!

---

Fair enough